### ISSUE TITLE
[Question] Include Add-On support for AE2 in personal build

### ISSUE THREAD

**Issue by viper2g1** [MightyPirates/OpenComputers#2249]
Sorry for asking in the issues page but I have a question - 

First of all, with your permission to compile at home for a home server, could you tell me how to reenable AE2 support in the mod? Is it as simple as allowing it to compile in build.gradle? Or would it be more involved? 

I know AE2 is still alpha but I'd like try none the less.

I'm not asking you to add support or anything officially, just information.

Thank you

**Comment by brisingraerowing** [comment]
A quick attempt to build it and I got hundreds of errors. It appears the AE2 API has changed majorly, so some major rework of the integration needs to be done.

**Comment by viper2g1** [comment]
Thanks for checking. Sounds like a lot of work needs to be done. Could you share how you manually compiled AE2 in?  I'd like to turn this into a learning experience and try to fix some errors.

**Comment by brisingraerowing** [comment]
I initially tried to copy how AE2Stuff by BDew does it, but couldn't get it to work, so I copied the jar to the libs directory and referenced it from there with `compile files("ae2-jar-name.jar")`.

A lot of errors were related to the compiler not being able to find classes, which aren't the hardest to fix (unless the class in question was removed).

I also just realized I was using the release jar, not the API jar, so I am trying the API jar now to see what happens (probably a lot of errors).
